docs/advanced-usage.rst

```diff
@@ -115,23 +115,27 @@ Recap: Asset Creation
     sent_tx = bdb.transactions.send(signed_tx)
     sent_tx == signed_tx
 
-Read the Creation Transaction from the DB
------------------------------------------
+Check if the Transaction was sent successfully
+-----------------------------------------------
 
-After a couple of seconds, we can check if the transaction was validated in a
+After a couple of seconds, we can check if the transaction was included in a
 block.
 
 .. code-block:: python
 
-    # Retrieve a block height
+    # Retrieve the block height
     >>> block_height = bdb.blocks.get(txid=signed_tx['id'])
 
-This will return the block height containing the transaction. If the transaction is not in any block then ``None`` is returned. If we want some more data we can use the block height to retrieve the block itself.
+This will return the block height containing the transaction. If the transaction is not in any block then ``None`` is
+returned. If it is ``None`` it can have different reasons for example the transaction was not valid or is
```

Comment:
In line 130 you write 'the transaction was not valid' and in line 131 you say 'was invalid'. Unless these have some semantic difference, I'd suggest using one consistent word choice everywhere, i.e. 'invalid' instead of 'not valid'